.github/workflows/e2e.yaml

```
@@ -2,44 +2,42 @@ name: End-to-End Tests
 on:
   workflow_dispatch:
   pull_request:
-concurrency: build-wsl
+concurrency: e2e
+
 jobs:
-  start_vm:
+  start-vm:
     strategy:
       max-parallel: 1
     runs-on: ubuntu-latest
     steps:
-      - uses: azure/login@v1
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Start Azure VM
+        uses: ubuntu/WSL/.github/actions/setup_vm/@ci-encapsulate-azure-wsl-309

```

This change will have to be made before merging:
```suggestion
        uses: ubuntu/WSL/.github/actions/setup_vm/@main
```

I didn't manage to make these track the current branch tho, so I hard-coded them to test the CI.

---

When tracking main, do you need `@main`? I didn’t need this [in this example](https://github.com/ubuntu/sync-issues-github-jira/blob/main/.github/workflows/example.yaml#L10). Note though that the event is different, being "gollum"

---

Good point, let me try it.